src/io/utilities/parseSBMLNotesField2012.m

```diff
@@ -63,15 +63,15 @@
     elseif (regexp(fieldStr,'^FORMULA'))
         formula = regexprep(strrep(fieldStr,'FORMULA:',''),'^(\s)+','');
     elseif (regexp(fieldStr,'^CHARGE'))
-        charge = str2num(regexprep(strrep(fieldStr,'CHARGE:',''),'^(\s)+',''));
+        charge = str2double(regexprep(strrep(fieldStr,'CHARGE:',''),'^(\s)+',''));
     elseif (regexp(fieldStr,'AUTHORS'))
         if isempty(citation)
             citation = strcat(regexprep(strrep(fieldStr,'AUTHORS:',''),'^(\s)+',''));
         else
             citation = strcat(citation,';',regexprep(strrep(fieldStr,'AUTHORS:',''),'^(\s)+',''));
         end
     elseif (regexp(fieldStr,'^Confidence Level'))
-        confidenceScore = regexprep(strrep(fieldStr,'Confidence Level:',''),'^(\s)+','');
+        [matches, tmpTokens] = regexpi(fieldStr, 'Confidence[ _]Level: (\w+)', 'match', 'tokens');
```

Comment:
Okay, if addReaction assumes it is a cell array, keep it as you suggested.
I would say this assumption doesn't make a lot of sense, and we should probably correct it when fixing model fields, but that's a future discussion.